WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -94,6 +94,7 @@ + (NSArray *)routes
     [[FBRoute POST:@"/wda/touchAndHold"] respondWithTarget:self action:@selector(handleTouchAndHoldCoordinate:)],
     [[FBRoute POST:@"/wda/doubleTap"] respondWithTarget:self action:@selector(handleDoubleTapCoordinate:)],
     [[FBRoute POST:@"/wda/pickerwheel/:uuid/select"] respondWithTarget:self action:@selector(handleWheelSelect:)],
+    [[FBRoute POST:@"/wda/pickerwheel/:uuid/selectvalue"] respondWithTarget:self action:@selector(handleWheelSelectValue:)],
```

Comment:
what is the difference between the previous endpoint and this one?

Reply:
We can just add another optional parameter to the above endpoint to avoid unnecessary duplication. I don't see a need for a separate endpoint there.